### DISCUSSION TITLE
The trap of lacking experience in advertising and knowledge of conversion rates

### DISCUSSION THREAD

**Discussion by goingsnginx** [sunflower-land/sunflower-land#2473] [General]
As a developer with experience in advertising,I've seen many developers fall into the trap of idealism, having little knowledge of operations and being unaware of concepts like ad placements and conversion rates. This often leads them to work hard on tasks with little impact.

I fear that the team may fall into this kind of misguided thinking or even obsession.

The team's ideal is to create a web3 game with pure gameplay that can achieve millions of daily active users. I respect this vision, and I think you're doing a great job, but the game is currently facing some difficulties.

In short, the conversion rate of new users is directly proportional to the price of SFL.

There were two changes in February:

1.The daily rewards put us in the top 10 of the DappRadar game rankings and on the first page of all dapp rankings, bringing significant exposure to the game. This means that we already have a solid foundation of nature traffic.
2.Matic contributed to the rise in the price of SFL.

Under the influence of these two factors, SunflowerLand gained 150-250 daily new natural users. If this situation persists, it would easily result in 5,000 new users in a month, and that's only with a relatively stable token performance.

However, with the decline in SFL price, the natural increase in new users from late February to March decreased significantly. Although we launched a referral campaign, its effect was minimal, as the price of SFL made it difficult to convert users.

This is not because everyone isn't trying hard to promote; it's because the conversion difficulty for new users in March was much higher than in February. At the same time, there are many distractions in the web3 ecosystem, leaving users with little time for a project with a continually falling token.

Worst of all, daily active user data has begun to decline in recent days, proving that many previously converted users haven't stayed. This is dangerous, as new user retention may not be ideal.

The root cause of all these issues is the instability of SFL's price – there's no doubt about it.

So, any marketing activities, even self-funded ad placements, would yield poor results under SFL's current performance because conversion is difficult, and retention is poor.

In my view, if no measures are taken to address SFL's price, any marketing efforts would be ineffective, and achieving millions of daily active users would be a pipe dream.

Axie didn't achieve a million daily active users on its own, and of course, SunflowerLand is much stronger than Axie in many aspects.

In the short term, I believe the only ways to increase the token's value are dynamic staking or listing on CEX. However, it seems the team won't undertake these measures for now, so I can only hope for breakthroughs in in-game resource consumption.

In summary, if we want the best marketing timing to acquire users, it must be when SFL's price is rising.

We don't lack exposure; we just need to improve the core element for converting users.

I still have faith in the team.
![image](https://user-images.githubusercontent.com/47945667/233615060-9240315b-9637-45dc-af73-bf351ebb138b.png)

**Comment by offerT** [comment]
Hope to see a response from the team

**Comment by future2back** [comment]
Also if the number of players is greater, the number of resources and SFL are greater, therefore, price is lower. For me, the price issue is because all players generate new resources and SFL everytime 24/7. For me, there is a lack of entertainment within the game, the multiplayer could bring that... while the user is generating resources could not leave the farm and participate in activities outside.

**Comment by goingsnginx** [comment]
If we face a conversion rate of only 0.5% for pure Web3 traffic, the conversion rate from Web2 might be as low as 0.05%.

This is the reality, so why would one think that Web2 users are more easily converted? Even though Binance had high expectations for the Hook, they still haven't managed to attract many qualified new users from Web2.

Based on a 0.05% conversion ability, ignoring retention factors, to achieve one million user registrations, 2,000,000,000 different individual Web2 ad exposures would be needed, which is close to 1/3 of the Earth's population seeing the advertisement to potentially realize this dream.

Therefore, the scientific approach is to reasonably use the existing Web3 exposure, improve the gameplay, token, and NFT potential, to significantly increase the conversion rate, rather than expecting an even lower conversion rate from Web2 users.

Guess the difference in conversion rates when the same game token has a performance of +50%, 0%, and -50% in the last 7 days. I believe there is a multiple difference between each, and the conversion rate between +50% and -50% may have a gap of more than 10 times.

Moreover, the largest traffic in the Web3 ecosystem is from CEX, especially Binance, which has  tens of millions of active users. The difficulty of converting users from CEX to Web3 is much lower than from Web2 to Web3, so relying on CEX for millions of active users is essential.

**Comment by adamhannigan** [comment]
Thanks for the in-depth analysis @goingsnginx 

We are aligned around conversion and the issues that Sunflower Land faces at the present. We are not going to endeavour traditional Web2 marketing strategies (e.g. paid advertising) as this doesn't align with Web3 conversion.

Right now we our marketing strategy is focussed around amplifying organic approaches and content - we are partnering with agencies that help with this content and creative approaches.

Engagement rate is closely correlated with SFL token price, more so than NFT prices. The past 6 months we haven't been focussed heavily on content (SFL burns), instead we decided to build out the core staples of gameplay (expansion, islands, landscaping, multiple bumpkins, plaza islands etc.). Right now we are shifting to content focussed, which means significant amount of collectibles/decorations which all require excess SFL and resource burns.

Staking SFL or any other financial services are not in our plans. These are obviously successful, but in the long term creates the wrong incentives and creates the wrong type of community that leads to an inevitable downfall. Players should hold items because they value the fun around them, instead of holding for some sort of reward.  

Hopefully over the coming months we can open source our analytics approach - it has a lot of useful data that someone like yourself would find valuable. Right now we are seeing great organic growth through our socials, Dapp Radar and natural search. Our funnel drop offs are largely around setting up a wallet and funding that wallet. We are definitely shifting more towards a growth mindset at the moment and doing data driven decisions. Any help is appreciated!

Let us know if you have more ideas 🤗

**Reply by goingsnginx** [reply]
Thank you, Adam, I really appreciate your skills as a game developer, and you're super amazing. When I first faced the dream of SunflowerLand with only gameplay achieving millions of daily active users, I never doubted it, and I thought the approach of catering to the grassroots players was filled with fairness and had great potential for the future.

However, until recently, I changed my mind.

Let me clearly analyze why there was such a huge drop this time, some reasons besides inflation:

1.Some people lost confidence and left during the minting event of Solar Flare Tickets. 
The problem with this event is:
The long task cycle is painful and exhausting. If someone works hard to catch up with the tasks but still can't get the ideal NFT, it's undoubtedly the most painful. If this process is very long, it will only remind them of the pain of not getting an NFT every day. This is different from grabbing NFTs by speed, where the pain of not getting an NFT only lasts a short time (although this method is not fair enough). In addition, the casual gambling approach of Treasure Island would be much better.Because before the outcome is determined, luck is a huge source of comfort in people's hearts. When a casual game makes people tired and in pain, it's not good.

2.Lack of competitiveness in good market conditions.
Indeed, some people leave because the market here is boring. If we want to be the world's number one Web3 game, we obviously need to have stronger competitiveness in a bull market, achieve a higher conversion rate than competitors, and it means that we need to have the ability to absorb a large amount of capital and attention in a bull market. We should not cause capital and players to leave due to a boring market in a bull market. In that case, SunflowerLand may only survive in a bear market, while underperforming in a bull market. Therefore, we might need some catalysts to accelerate the rotation of the economic system.

3.Lack of protection for major holders of SFL, causing some people to lose confidence or leave.
Q: Why would people buy SFL?
A: Because they believe the game has a chance for large-scale application in the future, triggering a rapid halving.

However, in fact, I gradually realized that SFL is the least protected. Most people are just selling SFL for profit. If a newcomer spends money, they might spend it on NFTs first, and then bring a large amount of SFL to sell. Yeah, everyone is producing and selling SFL. Who would buy SFL, and who would maintain the price of SFL? Perhaps only fools, these fools are constantly raising the price of SFL, paying for others' profits, but after a year, some people can't see the hope and leave.

Based on the above content, I would like to discuss another issue: whether the mere reliance on supply and demand can maintain a stable price for SFL. I think it's difficult unless there is a constant influx of new users. On the one hand, more than 50% of the Web3 audience may pursue maximum profit, which undermines the effectiveness of consumption scenarios like Treasure Island. On the other hand, it means the team needs to constantly introduce various NFTs to incentivize consumption, but will this create another problem: will the continuous stacking of NFT bonuses cause an explosion in production?

Therefore, I don't think that relying solely on in-game supply and demand can provide stable strength for SFL. So we think of those who paid for SFL, they came with hope, eventually paid for others' gains, and left with disappointment. Everyone dislikes them. Even those who make money from them mock and remind them, "This is a game, not a financial product." But those who can't make money will just leave nonchalantly, leaving only those fools who paid for SFL waiting for hope to come.

When I first saw the resources tilted towards the grassroots, I thought it was a great way to be fair. I think it's normal to make money from the rich to support the poor until I found out that when the balance tilts too far, everything could collapse. Yes, when you take money from the rich to please the poor with all your might, such an economic system will eventually be difficult to maintain. They may eventually leave in disappointment because it is an economic system with no protection for smart and economically capable people. Once they leave completely, the entire economic system will still spiral into death.

When I first saw the 100% pure gameplay, I thought it was really cool, and it was an amazing dream. I chose to support it without any doubt.

But recently, I've been thinking, is 100% pure gameplay the best path for Web3? Maybe a 9:1 or 8:2 ratio of gameplay to finance would be perfect.

Based on the above, I had an idea of dynamic staking:
The game provides SFL-denominated staking with APY ranging from 3% to 10%. The difference in returns comes from the level cap updates with each halving.

For example:
When we are at the 20 million+ token total stage with a level cap of 60, you need to reach level 60 to enjoy the highest APY. To encourage continuous leveling,
Level 59 APY 9%,
Level 58 APY 8%,
Level 50 APY 7%,
Level 45 APY 6%,
Level 40 APY 5%,
Level 30 APY 4%.
Level <30 APY 3%.

When we enter the 30 million+ token total stage, the level cap becomes 80.
Level 79 APY 9%,
Level 78 APY 8%,
Level 70 APY 7%.
Level <50 APY 3%.

When we enter the 40 million+ token total stage, the level cap becomes 100.
Level 99 APY 9%,
Level 98 APY 8%,
Level 90 APY 7%.
Level <60 APY 3%.

The reasons for doing this are:

1.Encourage people to engage deeply in the game. Only the highest level can enjoy the highest APY. This will drive a whole new Bumpkin trading market. When a wealthy person is optimistic about SFL and buys a large amount of SFL, if they want to quickly enjoy the highest APY, they can either buy a lot of resources to level up themselves or directly buy someone else's high-level Bumpkin. This will bring a strong driving force to the Bumpkin trading market and the resource market, encouraging countless people to be more enthusiastic about producing materials and leveling up Bumpkin.

2.With the subsidy of returns, buying SFL is no longer just paying for others' gains. Profits and risks can be hedged.

3.Control the liquidity of SFL, and possibly make the demand for SFL greater than the output, to achieve the basis for a positive cycle of SFL. Once SFL can achieve a stable, slow increase, the conversion rate of SunflowerLand will be guaranteed.

4.Accelerate the halving speed. In essence, it takes a portion of everyone's future gains to share current risks. Short-term inflation is exchanged for long-term deflation, so there's no need to worry about the impact of the SFL generated by these APYs on future inflation.

Of course, these rules for returns can be changed. At least in my calculations, even with 100 million active SFL tokens, this method would not have much impact on the overall ecosystem, only slightly increasing the halving speed. But the benefits might be obvious.

These are some of my views because, in my opinion, it's ideal for SFL to achieve a positive cycle, i.e., SFL slowly growing under restraint. This means that there needs to be a long-term stable relationship of buy-ins greater than sell-offs in the secondary market, and I think that relying solely on in-game supply and demand may not achieve this.

Finally, thank you, Adam, and thank you to the team. Although I lost some, I gained a lot.

**Reply by goingsnginx** [reply]
I suggest adding an email subscription feature, which reports weekly team updates and major event updates. It might provide some assistance in re-engaging inactive veteran players.

This includes two aspects:

1.Email
2.Choice of content language (mainstream languages would suffice)

The reason for incorporating language selection is because when dealing with regions where the average English proficiency is not high, the impact of English emails is considerably limited. Therefore, localizing the email content can significantly enhance the email's read rate. This might have originally required a high cost, but now we can use GPT-4 to translate the content, achieving a better translation effect than Google Translate.

To incentivize people to subscribe to the emails, rewards could be given upon first-time email linkage.

For users who have been banned from the game, they could be added to an email blacklist to prevent sending them emails. This is because if the email subscription is set within the game, they wouldn't be able to access the game to unbind it.

**Reply by YamiYamiKokiKopi** [reply]
> Thank you, Adam, I really appreciate your skills as a game developer, and you're super amazing. When I first faced the dream of SunflowerLand with only gameplay achieving millions of daily active users, I never doubted it, and I thought the approach of catering to the grassroots players was filled with fairness and had great potential for the future.
> 
> However, until recently, I changed my mind.
> 
> Let me clearly analyze why there was such a huge drop this time, some reasons besides inflation:
> 
> 1.Some people lost confidence and left during the minting event of Solar Flare Tickets. The problem with this event is: The long task cycle is painful and exhausting. If someone works hard to catch up with the tasks but still can't get the ideal NFT, it's undoubtedly the most painful. If this process is very long, it will only remind them of the pain of not getting an NFT every day. This is different from grabbing NFTs by speed, where the pain of not getting an NFT only lasts a short time (although this method is not fair enough). In addition, the casual gambling approach of Treasure Island would be much better.Because before the outcome is determined, luck is a huge source of comfort in people's hearts. When a casual game makes people tired and in pain, it's not good.
> 
> 2.Lack of competitiveness in good market conditions. Indeed, some people leave because the market here is boring. If we want to be the world's number one Web3 game, we obviously need to have stronger competitiveness in a bull market, achieve a higher conversion rate than competitors, and it means that we need to have the ability to absorb a large amount of capital and attention in a bull market. We should not cause capital and players to leave due to a boring market in a bull market. In that case, SunflowerLand may only survive in a bear market, while underperforming in a bull market. Therefore, we might need some catalysts to accelerate the rotation of the economic system.
> 
> 3.Lack of protection for major holders of SFL, causing some people to lose confidence or leave. Q: Why would people buy SFL? A: Because they believe the game has a chance for large-scale application in the future, triggering a rapid halving.
> 
> However, in fact, I gradually realized that SFL is the least protected. Most people are just selling SFL for profit. If a newcomer spends money, they might spend it on NFTs first, and then bring a large amount of SFL to sell. Yeah, everyone is producing and selling SFL. Who would buy SFL, and who would maintain the price of SFL? Perhaps only fools, these fools are constantly raising the price of SFL, paying for others' profits, but after a year, some people can't see the hope and leave.
> 
> Based on the above content, I would like to discuss another issue: whether the mere reliance on supply and demand can maintain a stable price for SFL. I think it's difficult unless there is a constant influx of new users. On the one hand, more than 50% of the Web3 audience may pursue maximum profit, which undermines the effectiveness of consumption scenarios like Treasure Island. On the other hand, it means the team needs to constantly introduce various NFTs to incentivize consumption, but will this create another problem: will the continuous stacking of NFT bonuses cause an explosion in production?
> 
> Therefore, I don't think that relying solely on in-game supply and demand can provide stable strength for SFL. So we think of those who paid for SFL, they came with hope, eventually paid for others' gains, and left with disappointment. Everyone dislikes them. Even those who make money from them mock and remind them, "This is a game, not a financial product." But those who can't make money will just leave nonchalantly, leaving only those fools who paid for SFL waiting for hope to come.
> 
> When I first saw the resources tilted towards the grassroots, I thought it was a great way to be fair. I think it's normal to make money from the rich to support the poor until I found out that when the balance tilts too far, everything could collapse. Yes, when you take money from the rich to please the poor with all your might, such an economic system will eventually be difficult to maintain. They may eventually leave in disappointment because it is an economic system with no protection for smart and economically capable people. Once they leave completely, the entire economic system will still spiral into death.
> 
> When I first saw the 100% pure gameplay, I thought it was really cool, and it was an amazing dream. I chose to support it without any doubt.
> 
> But recently, I've been thinking, is 100% pure gameplay the best path for Web3? Maybe a 9:1 or 8:2 ratio of gameplay to finance would be perfect.
> 
> Based on the above, I had an idea of dynamic staking: The game provides SFL-denominated staking with APY ranging from 3% to 10%. The difference in returns comes from the level cap updates with each halving.
> 
> For example: When we are at the 20 million+ token total stage with a level cap of 60, you need to reach level 60 to enjoy the highest APY. To encourage continuous leveling, Level 59 APY 9%, Level 58 APY 8%, Level 50 APY 7%, Level 45 APY 6%, Level 40 APY 5%, Level 30 APY 4%. Level <30 APY 3%.
> 
> When we enter the 30 million+ token total stage, the level cap becomes 80. Level 79 APY 9%, Level 78 APY 8%, Level 70 APY 7%. Level <50 APY 3%.
> 
> When we enter the 40 million+ token total stage, the level cap becomes 100. Level 99 APY 9%, Level 98 APY 8%, Level 90 APY 7%. Level <60 APY 3%.
> 
> The reasons for doing this are:
> 
> 1.Encourage people to engage deeply in the game. Only the highest level can enjoy the highest APY. This will drive a whole new Bumpkin trading market. When a wealthy person is optimistic about SFL and buys a large amount of SFL, if they want to quickly enjoy the highest APY, they can either buy a lot of resources to level up themselves or directly buy someone else's high-level Bumpkin. This will bring a strong driving force to the Bumpkin trading market and the resource market, encouraging countless people to be more enthusiastic about producing materials and leveling up Bumpkin.
> 
> 2.With the subsidy of returns, buying SFL is no longer just paying for others' gains. Profits and risks can be hedged.
> 
> 3.Control the liquidity of SFL, and possibly make the demand for SFL greater than the output, to achieve the basis for a positive cycle of SFL. Once SFL can achieve a stable, slow increase, the conversion rate of SunflowerLand will be guaranteed.
> 
> 4.Accelerate the halving speed. In essence, it takes a portion of everyone's future gains to share current risks. Short-term inflation is exchanged for long-term deflation, so there's no need to worry about the impact of the SFL generated by these APYs on future inflation.
> 
> Of course, these rules for returns can be changed. At least in my calculations, even with 100 million active SFL tokens, this method would not have much impact on the overall ecosystem, only slightly increasing the halving speed. But the benefits might be obvious.
> 
> These are some of my views because, in my opinion, it's ideal for SFL to achieve a positive cycle, i.e., SFL slowly growing under restraint. This means that there needs to be a long-term stable relationship of buy-ins greater than sell-offs in the secondary market, and I think that relying solely on in-game supply and demand may not achieve this.
> 
> Finally, thank you, Adam, and thank you to the team. Although I lost some, I gained a lot.

An interesting proposal, I wonder what the team thinks?

**Comment by goingsnginx** [comment]
@adamhannigan Hi,I believe you have noticed that recently, Discord's activity and enthusiasm for participation have been quite low. Although the active numbers may still appear decent, in my view, it is merely maintaining a relative balance, and the community's activity level may be even lower than the enthusiasm during the lowest point last year. Many players I know have started to lower their expectations and lose confidence. Although they are still habitually participating in the game, their enthusiasm has significantly decreased.

As I mentioned above, the SFL price affects the conversion rate,so the expectation for the price of SFL is another important factor influencing the overall data.

I started to think about a question: If there were no SunflowerFarm's accumulated fame and SFF's highest price of $5, could SunflowerLand have reached its current height?

I think it wouldn't have. From my perspective, SunflowerFarm's high popularity and SFF's high price gave me high expectations, which made it easy for me to convert to a SunflowerLand user. As for having confidence in the team, that's another story. If there were no SunflowerFarm, I probably wouldn't have paid attention to this game, and I wouldn't have known about your excellent team.

However, when SunflowerFarm's influence declines, the subsequent conversion and retention difficulties will undoubtedly increase. Most of them will not have as strong confidence and high expectations as we early players. If the recent performance is not good, they may have to say goodbye to the game.

The game is currently maintaining a fragile cycle. Although new content is continuously being released, the overall enthusiasm for participation is clearly declining. The expectations, enthusiasm, and investment of upper-level players are decreasing, and the earnings of lower-level players are diminishing. When assets no longer flow quickly, this is dangerous.

Despite my efforts to level up and expand my land, I can't find anyone to share these things with, which causes me to lose my competitive drive. Yes, leveling up has become a demotivating task for me now. On one hand, the cost-effectiveness of leveling up has started to decrease. On the other hand, the depressed community has made people stop comparing who has a higher level, and everyone begins to lose the sufficient reason to level up. I found that lower-level players are now generally stopping leveling up and focusing on mining, selling, and farming.

I'm starting to accept your insistence on non-financialized games, although I think they are too difficult. I believe you have also discovered how difficult it is to acquire new users without a high-expectation economic system. Not every game team has a high-popularity old project like SunflowerFarm.

Therefore, I think the current core factors are the following two:

1.SFL's price affecting the conversion rate (mainly affecting lower-level players)
2.SFL's price expectation affecting the conversion rate (mainly affecting upper-level players), long-term retention rate, and activity level

I don't think relying on consuming SFL to alleviate inflation can solve these problems. As for token halving, that requires more people to shorten the time, but we are currently bottlenecked in acquiring new users. Therefore, it might be more important to restore community activity first. At present, only listing on CEX can provide a sufficiently positive expectation.

**Comment by goingsnginx** [comment]
@adamhannigan Hi, there is a new game that is very similar to SunflowerLand: https://twitter.com/wofplatform. In fact, the team behind this game consists of former SunflowerLand players, and the game heavily draws inspiration from SunflowerLand's gameplay and systems while making some modifications.

This game quickly gained over 15000+ fans within a week, thanks to the whitelist activities promoted by several KOLs (the cost is relatively low, and some KOLs actively recommended it). Already, the whitelist for 3000 plots of land has exceeded 100u in price, and there are potentially tens of thousands of users interested in the game. Currently, there is a significant anticipation for a truly large-scale application game in the market, and the user base of this game can easily be converted from SunflowerLand players since the two types of games are very similar.

In fact, the entire Web3 world is eagerly awaiting a game that can truly achieve mass adoption. There are many potential users out there, and everyone is excitedly talking about the upcoming Gamefi Summer.

I understand that the lifecycle of this game may not be as long-lasting as SunflowerLand, but its market-oriented behavior makes it more likely to attract more users during a bull market. Moreover, these users were originally potential SunflowerLand players.

However, we must discuss the question of how to beat the competition and why users would choose this game over SunflowerLand. This involves tens of thousands of users and whose game they ultimately belong to.

I must say that the inflation issue has been addressed remarkably well, but it is not enough. Many resource-consuming players do not lack SFL, and they will not need to buy SFL for a long time. SFL still needs an external force that actively buys in to drive up the price, at least to show new players some profits, in order to thrive in the competition.

**Reply by Matarener** [reply]
I would even say that this is a very dangerous competitor for sfl. It almost completely copies the mechanics of sfl from beta, and at the same time promises players to earn money. I know many sfl players who have already begun to actively play wof and so far they are satisfied with everything. Also, do not forget about the fomo aspect (in the game you can craft many NFTs early enough, the analogues of which in SFL cost a lot). Plus, their project uses a very advanced two-token economy, as in axies, for example, they promise listing and staking of a token, and we remember that axies has shown itself quite well and is still alive

**Comment by goingsnginx** [comment]
@adamhannigan From what I know, there are some KOLs who focus on airdrops and follow SunflowerLand on Twitter. These KOLs have a large number of active users, although the quality of these users may not be high, they are highly engaged. This provides a good foundation for dissemination.

I think Twitter advertising can be tried initially with a zero-cost approach to see if it can be retweeted by some KOLs in the context of organic dissemination and generate secondary dissemination. Based on this premise, I believe the campaign needs to be special, such as coordinating with a new season, offering a one-week free registration for SunflowerLand or limited 100k free new accounts, requiring users to follow the Twitter account, retweet specific tweets, join Discord, and verify their participation to obtain a free account. galxe.com should be able to meet these types of task requirements.

The purpose of such a campaign is to try to spread SunflowerLand through Twitter and be retweeted by KOLs focused on airdrops, thereby attracting a large number of new users. Although there may be many multiple accounts involved, the floor price of the land is already close to 0.2 MATIC, which is almost the same as free.

This is an attempt, a test that can yield interesting data.

Of course, I noticed that you are integrating the login with OKX. It would be even better if you could collaborate with OKX officials to organize a campaign that encourages users to use the OKX wallet to play SunflowerLand.

**Reply by Jericho-rpc** [reply]
Agree with your point of view, rekindling this game must require the growth of new users, and the growth of new users must require new game activities to attract them (fun and rewards). I think the activities that should be launched should allow all players to stand on the same starting line Fair competition, activities should be game-like and interesting, such as fishing activities, reward exchange for artwork NFT, enhancement and upgrade of different fishing rod quality (game nature with failure probability), continuous burning of sfl and minerals, etc.

**Comment by poteitouweb3** [comment]
@adamhannigan  if I may contribute, just focus on the web2 audience. you are doing an amazing job in simplifying the use of tokens/crypts and if we add that to a really playable game, we'll have an amazing audience here that will not worry acidly about rewards.

I had commented in the coder's chat but i reinforce: We VERY much need the MMO mechanics to expand to the whole game, because then it is easier to produce content, livestreams and so on.

I dream of the day I will login Sunflower Land, spawn with my character, equip my axe, walk to the trees, collect them, then the same for my ores, a sickle to harvest my crops, then a bag of seeds to plant, just like it is today in Stardew Valley... It's the kind of routine that today takes less than 1 minute but with playable mechanics would generate a gameplay of at least 10 minutes to do everything in the land. This is the biggest and best marketing for the game.
